schemas/Case.yaml

```diff
@@ -9,18 +9,10 @@ required:
 properties:
   id:
     $ref: Id.yaml
-  eaches:
+  items:
```

Comment:
I'm not convinced that item vs eaches rename is actually the correct way to describe something. I would expect an item to be a single product, but an each might refer to a package of products sold as a single unit, thoughts on this @1e1f

Reply:
I'm ok with this.